examples/todoapp/.github/workflows/fastlane_build_and_deploy_android_app.yaml

```diff
@@ -0,0 +1,113 @@
+name: "Custom build and deploy Android app"
```

Comment:
The github actions workflow files must be in the repository root

Reply:
They are,
I have also removed these files from todoapp root directory

Reply:
https://github.com/Prime-Holding/rx_bloc/blob/a8f7fc85d1c4cc04df31b52f72ab5333d872e512/.github/workflows/todoapp_build_and_deploy_app.yaml